cmake/FindArqma.cmake

```diff
@@ -44,7 +44,7 @@ foreach (l ${LIBS})
 	find_library(Arqma_${L}_LIBRARY
 		NAMES ${l}
 		PATHS ${CMAKE_LIBRARY_PATH}
-		PATH_SUFFIXES "/src/${l}" "/src/" "/external/db_drivers/lib${l}" "/lib" "/src/crypto" "/contrib/epee/src" "/external/easylogging++/" "/external/${l}"
+		PATH_SUFFIXES "/src/${l}" "/src/" "/external/db_drivers/lib${l}" "/lib" "/src/crypto" "/contrib/epee/src" "/external/easylogging++/" "/external/randomarq"
```

Comment:
${l} means all libraries and dont need to point exact one

Reply:
In that case easylogging++ can be removed as well as randomarq 
(it's not issue)